<p>&lt;p&gt;The study of &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Genome&quot;&gt;Genome&lt;/a&gt;s of organisms&lt;/p&gt;&lt;p&gt;&lt;a class=&quot;tc-tiddlylink-external&quot; href=&quot;https://www.wikiwand.com/en/Genomics&quot; rel=&quot;noopener noreferrer&quot; target=&quot;_blank&quot;&gt;https://www.wikiwand.com/en/Genomics&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;a class=&quot;tc-tiddlylink-external&quot; href=&quot;https://genome-euro.ucsc.edu/cgi-bin/hgGateway?hgsid=218799371_GrzgaLCTgLS4ixN0o64W7m83y2na&quot; rel=&quot;noopener noreferrer&quot; target=&quot;_blank&quot;&gt;https://genome-euro.ucsc.edu/cgi-bin/hgGateway?hgsid=218799371_GrzgaLCTgLS4ixN0o64W7m83y2na&lt;/a&gt;&lt;/p&gt;&lt;h2 class=&quot;&quot;&gt;&lt;u&gt;Comparative genomics&lt;/u&gt;&lt;/h2&gt;&lt;p&gt;Has uses in &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Biochemical%20genetics&quot;&gt;Biochemical genetics&lt;/a&gt;, &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Population%20genetics&quot;&gt;Population genetics&lt;/a&gt;, etc..&lt;/p&gt;&lt;hr&gt;&lt;p&gt;&lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Bioinformatics&quot;&gt;Bioinformatics&lt;/a&gt;&lt;/p&gt;&lt;p&gt;&lt;a class=&quot;tc-tiddlylink tc-tiddlylink-missing&quot; href=&quot;#Haplotype%20inference&quot;&gt;Haplotype inference&lt;/a&gt; – Infering the &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Haplotype&quot;&gt;Haplotype&lt;/a&gt;s (the two homologous chromosomes) from genomic data.&lt;/p&gt;&lt;p&gt;&lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#DNA%20sequencing&quot;&gt;DNA sequencing&lt;/a&gt; just gives all the base pairs at each position, but doesn't give which belong together in a single chromosome/haplotype.&lt;/p&gt;&lt;p&gt;Heterozygous &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Single-nucleotide%20polymorphism&quot;&gt;SNP&lt;/a&gt;s are the ones which are non-trivial to assign to the chromosome/haplotype.&lt;/p&gt;&lt;ul&gt;&lt;li&gt;Clark's algorithm: a parsimony approach. Try to minimize the different number of haplotypes,&lt;/li&gt;&lt;li&gt;Statistical model approach. Find all potential haplotypes consistent with data. Model assigns a population frequency to each haplotype. If we know these frequencies we can assign the most probable haplotypes to each individual. &lt;em&gt;Missing data problem&lt;/em&gt; –&amp;gt; &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#EM%20algorithm&quot;&gt;EM algorithm&lt;/a&gt;&lt;/li&gt;&lt;li&gt;Bayesian methods. Allow haploids which are different, but similar to influence each other's likelihood. &lt;a class=&quot;tc-tiddlylink tc-tiddlylink-resolves&quot; href=&quot;#Hidden%20Markov%20model&quot;&gt;Hidden Markov model&lt;/a&gt;.&lt;/li&gt;&lt;/ul&gt;</p>